spec/controllers/batch_contexts_controller_spec.rb

```diff
@@ -63,7 +63,7 @@
           bc = assigns(:batch_context)
           expect(bc.project_name).to eq 'Multimedia'
           expect(bc.content_structure).to eq 'simple_image'
-          expect(bc.processing_configuration).to eq 'default'
+          expect(bc.processing_configuration).to eq 'filename'
           expect(bc.staging_location).to eq 'spec/fixtures/multimedia'
```

Comment:
these changes are because the default batch_context factory now creates image content type with filename processing configuration (because "image / default" combo is now invalid)